jobs/config.json

```diff
@@ -4773,7 +5116,8 @@
       "--test=false", 
       "--kubemark", 
       "--timeout=60m", 
-      "--extract=ci/latest"
+      "--extract=ci/latest", 
+      "--check-leaked-resources=false"
```

Comment:
True, but do you mind if I handle that in a later commit? This is easier to understand (and migrate) since we are switching the default from checking resources (if you're on GCE or GKE) to not checking resources.

Once this PR sticks then we can make that change?